external/harvestPackages/harvestPackages.depproj

```diff
@@ -4,143 +4,167 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.6</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CSharp">
```

Comment:
Consider adding a comment here that says build /t:OutputLatestStablePackages to update.  Also consider moving this to a targets file and having the target just WriteLinesToFile to it.